gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Tue Sep 20 16:08:53 CEST 2016
+#Sat Mar 20 17:49:01 WET 2021
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip
```

Comment:
Why would we update it only to `6.5` given that latest version is `6.8.3`? Also I'd recommend to keep `all` to have sources available.